pages/phases/job-analysis/defining-proficiency-levels.md

```diff
@@ -6,48 +6,91 @@ title: Defining Proficiency Levels and Required Experience
 
-- **Scale of activity:** The applicant did something at a small organization _-vs-_ did something at a large organization.
-- **Seniority:** The applicant did something as part of a group _-vs-_ lead the group that did something.
+- **Complexity:** The applicant can manage straightforward workflows _-vs-_ can manage highly complex workflows.
 - **Level of detail:** The applicant can explain a general concept _-vs-_ can discuss the concept in detail.
+- **Scale of activity:** The applicant did something at a small organization _-vs-_ did something at a large organization.
+- **Seniority/independence:** The applicant did something as part of a group _-vs-_ led the group that did something.
 
-### Setting the required proficiency level
+When defining proficiency levels, you have two options.
+
+### Two levels
+
+Use two levels if your announcement is only for one GS grade and you want to save time during job analysis by defining the minimum required levels: the minimum required level ("meets") and one level above that ("exceeds") to establish categories as required by category rating rules. If the hiring action was for a GS-13 position, the “meets” level would mean the applicant needs 52 weeks experience at that level in order to qualify for GS-13. If your hiring action is for two GS grades, you will need at least three proficiency levels.
+
+#### Example (for an analytical competancy)
+
+* **GS-13 Meets:** Breaks problems down into component parts. Displays repeated
+experience in qualitative and quantitative analysis. Defines product or project metrics
+beyond the basics and ties these metrics to decisions.
+* **GS-13 Exceeds:** Brings analytical thinking to everything they do, and has a range and depth of
+experience doing so. Anticipates the need for metrics and analysis early in the product
+and project discovery and design process, and carries metrics through iterations.
+
+### Four levels
+
+Use four levels when you may want to leverage this work for other grades in the future; the same set of competencies and proficiencies can be used for different grade levels because roles that require more seniority set the required proficiencies higher than junior roles. It may also help to define four levels before deciding which level is appropriate for the grade in question. If you use four levels, we recommend these titles for each level:
+
+1. Novice
+2. Competant
+3. Advanced
+4. Expert
+
+Once proficiency levels are defined, HR and the SMEs will decide at which level you would require applicants to have one year of experience in order to qualify for the grade in question. For example, they made decide that to qualify for a GS-13, applicants must have one year of experience at the “Advanced” level. See this example:
+
+#### Example (for an analytical competancy)
+
+* **Novice:** Unable to break down problems or only talks through problems at a high
+level. Does not make data-driven decisions, or actively eschews data in decision-making.
```

Comment:
```suggestion
level. Does not make data-driven decisions, or actively eschews collaboration in decision-making.
```

Reply:
rejecting this because the collaboration text is different across the board